mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -20,7 +20,7 @@ CLASS sg net/minecraft/entity/Entity
 	FIELD N prevRenderY D
 	FIELD O prevRenderZ D
 	FIELD P stepHeight F
-	FIELD Q noClip Z
+	FIELD Q ignoreCollision Z
```

Comment:
`noClip` is pretty widely understood.
Does this ignore collision with Entities as well as Blocks?